lib/fileProcessor.js

```diff
@@ -0,0 +1,59 @@
+const moment = require('moment');
+class FileProcessor {
+  constructor(store, files) {
```

Comment:
hmmm, honestly don't know what I was looking at when I wrote that comment but do recall it was on GitHub in some file.  You are right, the GitHubProcessor constructor does not have options in the  signature.  Ignore my comment.